src/flightRecorder.cpp

```diff
@@ -1382,11 +1382,9 @@ Error FlightRecorder::startMasterRecording(Arguments& args, const char* filename
 
     jobject jfilename = env->NewStringUTF(filename);
     jobject jsettings = args._jfr_sync == NULL ? NULL : env->NewStringUTF(args._jfr_sync);
-    int event_mask = (args._event != NULL ? 1 : 0) |
-                     (args._alloc >= 0 ? 2 : 0) |
-                     (args._lock >= 0 ? 4 : 0) |
-                     (args._latency >= 0 ? 8 : 0) |
-                     ((args._jfr_options ^ JFR_SYNC_OPTS) << 4);
+
+    int event_mask = make_event_mask(args) |
+                     ((args._jfr_options ^ JFR_SYNC_OPTS) << (EVENT_MASK_SIZE - 1));
```

Comment:
Why not `JFR_OPTS_SHIFT`?

Reply:
I changed both places to use the same variable naming: 8c8cfae74661ccc23706d0e612e98c35cc9b0b70